pkg/discovery/discovery.go

```diff
@@ -361,23 +361,34 @@ func (d *Discoverer) discoverVolumesAtPath(class string, config common.MountConf
 				discoErrors = append(discoErrors, fmt.Errorf("path %q of filesystem mode cannot be used to create block volume", filePath))
 				continue
 			}
-			// TODO(mauriciopoppe): skipping the mountpoint verification in Windows, unlike Linux there's no /proc/mounts file to check
 			if runtime.GOOS == "windows" {
 				// the path to use is the one sent to CSI Proxy and must be in the context of the host
 				// not the container, filePath is in the context of the container
-				volumePathInHost := filepath.Join(config.HostDir, file)
-				capacityByte, err = d.VolUtil.GetFsCapacityByte(volumePathInHost)
+				volumePathInHostContext := filepath.Join(config.HostDir, file)
+
+				// in Windows a path is likely a mount point if it's a symlink pointing to a path that exists
+				isLikelyMountPoint, err := d.VolUtil.IsLikelyMountPoint(volumePathInHostContext)
+				if !isLikelyMountPoint || err != nil {
+					discoErrors = append(discoErrors, fmt.Errorf("path %q is likely not a mount point: %v", filePath, err))
```

Comment:
For the error message, can we make it "is not a symlink" since in Windows mount points are a different concept? A user seeing the error message should be able to easily understand that they need to create a symlink.

Reply:
The error looks like this now:

```
I1208 06:09:06.301871    7108 discovery.go:411] Found new volume at host path "c:\\mnt\\disks\\ssd0" with capacity 1056894976, creating Local PV "local-pv-2066695
9", required volumeMode "Filesystem"
I1208 06:09:06.326949    7108 cache.go:55] Added pv "local-pv-20666959" to cache
I1208 06:09:06.327831    7108 discovery.go:445] Created PV "local-pv-20666959" for volume at "c:\\mnt\\disks\\ssd0"
E1208 06:09:06.327831    7108 discovery.go:221] Failed to discover local volumes: 1 error(s) while discovering volumes: [path "c:\\mnt\\disks\\fake-ssd1" is not a valid mount point: hostPath "c:\\mnt\\disks\\fake-ssd1" is not a symlink]
```